### ISSUE TITLE
Can't send email out to SMTP

### ISSUE THREAD

**Issue by mkz71** [tomav/docker-mailserver#150]
Not sure if I did something wrong.
Can't send out email to SMTP.
I tried it on Thunderbird with "None" Connection security for  both IMAP and SMTP.
I can receive incoming mail but not going out.

**Comment by tomav** [comment]
Hi @mkz71, what the logs say when you try to send an email?

**Comment by mkz71** [comment]
Where do I check the logs?

I did some testings and it seems that I have to change SMTP server setting to make it work.
I changed Connection security from "None" to "STARTTLS" before I can send it.
Is this correct setting?

Another issue I have is if I send the mail when I connect to vpn service, it won't send the mail.
I got connection timed out to SMTP server.
If I disable the vpn connection, I can send it with the setting I mentioned above.
